### ISSUE TITLE
Dashboard variable, timepicker, and refresh settings not preserved correctly after navigating away and back in versions 11.3.x and 11.4.0

### ISSUE THREAD

**Issue by krzysion** [grafana/grafana#97699]
### What happened?

Since versions 11.3.x and 11.4.0, the behavior of dashboard variable settings, timepicker settings, and refresh intervals has changed.

When I change a variable or select a different time range on the timepicker and then navigate to another dashboard and back, the saved settings for the variables and timepicker are not loaded. Instead, the last manually selected settings are used. This also breaks navigating to a dashboard from another dashboard using links, as the saved variables and timepicker settings in the linked dashboard are ignored in favor of the settings from the previous visit.

This behavior persists until the browser is closed.

### What did you expect to happen?

When navigating back to a dashboard, the saved variable and timepicker settings should be loaded.
When navigating to a dashboard via a link with specified variables and time range, those settings should be applied.

### Did this work before?

All before 11.3.0

### How do we reproduce it?

Open a dashboard with variables and a timepicker.
Change a variable or the time range.
Navigate to another dashboard.
Navigate back to the original dashboard.
Observe that the previously selected variable and timepicker settings are applied instead of the saved settings.

### Environment (with versions)?

Grafana: 11.3.x 11.4.0

### Grafana platform?

None

**Comment by AsaadNA** [comment]
Yes, this is a major issue and affecting us in production enviroments for our support team.

**Comment by trajche-manolev** [comment]
We are also facing this issue after upgrading from Grafana 11.0.0 to 11.6.0. The dashboard retains the last used time range instead of reverting to the default time range upon reopening. This is causing usability issues for our team.

It seems that this regression was introduced with Scenes-Powered Dashboards in Grafana 11.3.0.

**Comment by krzysion** [comment]
Hi,
Just checking in on this issue. Any updates on the status or triage? Happy to provide more info if needed!
Also curious if there's any chance this might get looked at for the Grafana 12 release, or is it too early to say?

Thanks for all the great work on Grafana!

**Comment by torkelo** [comment]
This change is by design, when coming back (either via browser history or other link) the thinking is that you mostly likely want the same variable and time range state you had previously

**Comment by trajche-manolev** [comment]
Frankly speaking, I don’t fully understand the intent behind preserving the previous state. For many years, Grafana dashboards have always reverted to their configured default time range upon opening—and that behavior worked well for us. It seems that what’s now considered the “most likely” behavior wasn’t actually the norm or expected by most users until Grafana 11.3.0.

Is there any way to disable this behavior without turning off Scenes-Powered Dashboards entirely? Having dashboards open with the default time range is crucial for our team’s workflows, and the current behavior is causing confusion and inconsistencies.

**Comment by stevmei** [comment]
Still an issue with 11.6.x

**Comment by Damian92666** [comment]
Hello!

When I switch to version 11.3.0, my dashboards stop working as they should. The problem lies in the variables added to the URL.

**Grafana version 11.3.0, URL - Level 1:**
http://grafana.domain.com/d/T23vdVn/monitoring-l1?orgId=1&from=now-15m&to=now&timezone=utc&var-datasource=W_5SFDXsz&var-customers=$__all&var-env_types=$__all&var-host_names=$__all&var-component=$__all&refresh=3m

**Grafana version 11.2.10, URL - Level 1:**
http://grafana.domain.com/d/T23vdVn/monitoring-l1?orgId=1&refresh=3m

So when I switch to monitoring-l2, I can not read variables.

**Grafana version 11.3.0, URL - Level 2:**
http://grafana.domain.com/d/z37657XWQ/monitoring-l2?var-host_names=XXX_TEST&orgId=1&from=now-5m&to=now&timezone=utc&var-datasource=W_5SFDXsz&var-customers=&var-env_types=&var-env_name=&var-component=$__all&var-application=$__all&refresh=3m

**Grafana version 11.2.10, URL - Level 2:**
http://grafana.domain.com/d/z37657XWQ/monitoring-l2?var-host_names=XXX_TEST&orgId=1&refresh=3m

**EDIT:**

**I found a solution, maybe will be helpful for someone:**

**In Grafana 11.3.0, the "Constant" variable type is being replaced by the "Custom" variable type.**

Regards,
Damian

**Comment by Damian92666** [comment]
Hello!

Unfortunately changing the type from "Constant" to "Custom" didn't fix all my problems.

When I switch between dashboards using a Data Links, I notice that the data on the second dashboard panel loads more slowly. Sometimes it doesn't load at all, and instead of actual values, I see variables (e.g., $customer, $env_name). Has anyone experienced a similar issue?

Best regards,
Damian

**Comment by trajche-manolev** [comment]
We previously worked around this by disabling Scenes in Grafana, expecting a proper solution for preserving time ranges:
`dashboardScene = false 
publicDashboardsScene = false  
dashboardSceneForViewers = false  
returnToPrevious = true  `

However, no solution has been provided so far. Moreover, in Grafana 13 these options no longer work, removing our ability to control dashboard state behavior.

For our production use case (screen-based dashboards and navigation), dashboards must always open with a predefined/default time range.

The current behavior—persisting the last session state—breaks navigation and leads to inconsistent results. This is a functional regression, not just a UX improvement and "mostly likely behavior".

Is there any supported way in Grafana 13 to:
- disable state persistence, or
- force dashboards to respect default parameters?
